aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -209,6 +209,309 @@ SignalR hub methods now support injecting services through dependency injection
+
+For more information, see the following articles:
+
+* <xref:blazor/tooling?view=aspnetcore-7.0>
+* <xref:blazor/project-structure?view=aspnetcore-7.0>
+
+### Blazor custom elements no longer experimental
+
+The [`Microsoft.AspNetCore.Components.CustomElements`](https://www.nuget.org/packages/microsoft.aspnetcore.components.customelements) package for building [standards based custom elements](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements) is no longer experimental.
+
+For more information, see <xref:blazor/components/index?view=aspnetcore-7.0#blazor-custom-elements>.
+
+<!--
+
+    IN PROGRESS
+
+### Improve prerendering and authentication experience in Blazor
+
+[27592](https://github.com/dotnet/aspnetcore/issues/27592)
+
+-->
+
+### Bind get/set/after modifiers
+
+New modifiers, `@bind:get`/`@bind:set`/`@bind:after`, allow you to customize how Blazor's `@bind` attribute directive reads and writes bound values.
+
+<!--
+
+    NOTE
+    
+    `@bind:get`/`@bind:set` are only covered in an example for binding across more than two components. 
+    There isn't textual coverage outside of that section or dedicated examples for these.
+
+-->
+
+For more information, see the following content in the *Data binding* article:
+
+* [`@bind:after="{EVENT}"` content](xref:blazor/components/data-binding?view=aspnetcore-7.0) at the end of the article's introductory guidance.
+* The [Bind across more than two components](xref:blazor/components/data-binding?view=aspnetcore-7.0#bind-across-more-than-two-components) section demonstrates the use of `@bind:get`/`@bind:set` syntax.
+
+Additional information:
+
+* [Design proposal: Bind get/set/after modifiers (dotnet/aspnetcore #39837)](https://github.com/dotnet/aspnetcore/issues/39837)
+* [Support `@bind:get`, `@bind:set`, `@bind:after` (dotnet/razor-compiler #70)](https://github.com/dotnet/razor-compiler/pull/70)
+
+> [!WARNING]
+> Final design changes prior to the general release might not be reflected in the initial design described by the product unit's GitHub issue and pull request. For the latest information on this feature, see the preceding cross-links to the official Blazor documentation.
+
+<!--
+
+    IN PROGRESS
+
+### Enable integration with third-party assets management solutions
+
+[38445](https://github.com/dotnet/aspnetcore/issues/38445)
+
+-->
+
+<!--
+
+    HOLD
+    
+    This is a tooling update for what Steve says is a rare situation.
+    No doc updates were made to cover this, but we could include something here on it.
+    The following is a bit draft content for it.
+    
+### Components don't reset their received parameters during Hot Reload updates
```

Comment:
This was fixed in .NET 7, but it was more of a bug fix. I don't think we're going to list every improvement to hot reload support here. So I think this can be removed from the what's new list.

Reply:
The title of the section doesn't indicate correctly that the content of this explains that it was fixed.

I'll recast it all to that list ☝️ of improvements, and I'll add this one to the list (in a summarized line or two).